### PR TITLE
Footer Logo ammends

### PR DESCRIPTION
**Pull request by MazOneTwoOne**
Related to: https://github.com/theglobechurch/globe-static/issues/2

### CODE REVIEW

**Comment by jamesdoc** [comment]
Reviewing this; Realised that the logo that I gave you just wasn't going to behave properly with `rotate-90` – I want the logo snug against the right edge, so I've pushed an updated logo where the rotation is baked into the graphic (and the circle is cropped too) to the branch.

It's now sitting like this:

Screenshot 2022-11-24 at 07 14 59

A couple of minor tweaks that would be good to resolve here:

1. To get the logo where it is I `flex`ed the whole `<footer>`. This means that the top border extends too far now :( Can you move that down onto the `nav` 
2. The `flex` now means that the `nav` doesn't require the max-width on it.

With these two in, I think that we can swash and merge 👍

File: src/_includes/blocks/footer.njk [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
-        <li><a  href="/what-we-believe">What we believe</a></li>
-        <li><a  href="/affiliations">Affiliations</a></li>
-        <li><a  href="/jobs">Jobs</a></li>
-        <li><a  href="/policies">Policies</a></li>
-      </ul>
+    <nav>
+      <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-6 lg:grid-cols-6 gap-5 md:max-w-[75%]">
+        <ul>
+          <li><a class="text-base " href="/about">About</a></li>
+          <li><a  href="/the-globe-church">The Globe Church</a></li>
```
Comment (jamesdoc) [inline src/_includes/blocks/footer.njk:12]: In all these links there is a weird extra space between the `a` and the first property (eg `href`). Would be good to resolve that for all of the items.

**Review by jamesdoc — CHANGES_REQUESTED** [review]

**Comment by MazOneTwoOne** [comment]
- Top border adjusted, still used a max width to make sure it didn't overlap with logo. Looks like this now;
Screenshot 2022-11-24 at 22 24 50


- spaces between `a` and `href` now formatted.

**Review by jamesdoc — APPROVED** [review]
Looks good; please squash and merge 👍

**Pull request merged**